.gitlab/ci/on-push.yml

```diff
@@ -160,10 +168,16 @@ create-instances:
       fi
     - !reference [.open-ssh-tunnel]
     - python3 ./Tests/scripts/wait_until_server_ready.py "$INSTANCE_ROLE"
+    - |
+      # workaround for the hard-coded value in the sdk
+      cp "$ARTIFACTS_FOLDER/env_results.json" "./artifacts/env_results.json"
+      cp "$ARTIFACTS_FOLDER/filter_file.txt" "./artifacts/filter_file.txt"
+
```

Comment:
This will be removed after the `demisto-sdk` is updated?

Reply:
yes